dash_bio/package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Dash components for bioinformatics",
   "main": "build/index.js",
   "scripts": {
+    "start": "webpack-serve ./webpack.serve.config.js --open",
```

Comment:
What's the story behind this line?

Reply:
oups I forgot to delete it, done in 0c02ed994d173b9457f83025f3e285e8b701427e